perf/jet_test_nfailures.jl

```diff
@@ -1,16 +1,13 @@
 import Random
 Random.seed!(1234)
 import ClimaAtmos as CA
+import YAML
 
 include("common.jl")
 
-length(ARGS) < 2 && error("Usage: benchmark.jl <target_job> <job_id>")
-target_job = ARGS[1]
-job_id = get(ARGS, 2, target_job)
-
-config_dict =
-    target_job != "default" ? CA.config_from_target_job(target_job) :
-    CA.default_config_dict()
+length(ARGS) != 1 && error("Usage: benchmark.jl <config_file>")
```

Comment:
```suggestion
length(ARGS) != 1 && error("Usage: jet_test_nfailures.jl <config_file>")
```
Or `basename(@__FILE__)`